### PR TITLE
Brand management

### PR DESCRIPTION
**Pull request by Cell5**
intermediate Brand management implementation

### CODE REVIEW

File: src/components/BrandManagementComponents/BrandList.vue [hunk 1]
```diff
@@ -58,10 +54,11 @@ export default {
 </script>
 
 <style scoped>
-       .resultsContainer {
+    .resultsContainer {
         font-size: 10px;
```
Comment (epsotto) [inline src/components/BrandManagementComponents/BrandList.vue]: Change this to 12px. 10px is too small.

File: src/components/BrandManagementComponents/BrandList.vue [hunk 2]
```diff
@@ -58,10 +54,11 @@ export default {
...
     }
-    .cell{
+
+    .cell {
```
Comment (epsotto) [inline src/components/BrandManagementComponents/BrandList.vue]: If this class is not used, please remove.

File: src/components/BrandManagementComponents/SearchField.vue [hunk 3]
```diff
@@ -17,21 +17,22 @@
...
+    }
+
+    .btn {
+        height: 35px;
+        font-size: 15px;
+        margin-left: 10px;
+        width: 100px;
+    }
+    .text {
+        padding-top: 20px;
```
Comment (epsotto) [inline src/components/BrandManagementComponents/SearchField.vue]: Add font size 12px here.

File: src/components/BrandManagementComponents/BrandList.vue (hunk 1 above)
Comment (Cell5) [reply src/components/BrandManagementComponents/BrandList.vue]: done

File: src/components/BrandManagementComponents/BrandList.vue (hunk 2 above)
Comment (Cell5) [reply src/components/BrandManagementComponents/BrandList.vue]: done

File: src/components/BrandManagementComponents/SearchField.vue (hunk 3 above)
Comment (Cell5) [reply src/components/BrandManagementComponents/SearchField.vue]: component has been removed

**Pull request merged**